apps/fhir/authentication.py

```diff
@@ -18,10 +18,4 @@ def extract_username(auth, auth_prefix="SLS"):
     # raises UnicodeDecodeError
     auth_string_decoded = b64_decoded.decode(encoding)
 
-    return convert_sls_uuid(auth_string_decoded)
-
-
-def convert_sls_uuid(id):
```

Comment:
this overwrites the change in https://github.com/CMSgov/bluebutton-web-server/commit/176b118fb3467fe6dc424f40bc117d814714d429